pkg/kotsadm/kotsadm.go

```diff
@@ -16,6 +16,8 @@ import (
 
 var timeoutWaitingForKotsadm = time.Duration(time.Minute * 2)
 
+const KotsadmKey = "app.kubernetes.io/name"
```

Comment:
What other label names could we use here, that is more "kots" like, less "kubernetes"?

Reply:
"kots.io/control-plane=1"

Reply:
decided on "kots.io/kotsadm=true" on code review call